### ISSUE TITLE
[BUG] Connecting closed

### ISSUE THREAD

**Issue by KazeDevID** [WhiskeySockets/Baileys#929]
![Screenshot_20240716-150345](https://github.com/user-attachments/assets/4aef8576-849e-40ec-9187-b7f342668eb0)

**Comment by Guru322** [comment]
**same** 
428 while sending message to a group 
Dm works fine

**Comment by KazeDevID** [comment]
> 428 isBoom: true is your code!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

I've re-scanned several times, but after about 1 hour the error is like that again, do you have any suggestions??

**Comment by luispimenta** [comment]
Here exactly the same thing happens, but out of 40 groups only 4 have this problem, I can send a scheduled message or not, and the same 4 groups always don't receive it, giving the error 428, all the others work perfectly

[my issue](https://github.com/WhiskeySockets/Baileys/issues/914)

**Comment by KazeDevID** [comment]
> Di sini hal yang persis sama terjadi, tetapi dari 40 grup hanya 4 yang memiliki masalah ini, saya dapat mengirim pesan terjadwal atau tidak, dan 4 grup yang sama selalu tidak menerimanya, memberikan kesalahan 428, semua yang lain bekerja dengan sempurna
> 
> [Masalah saya](https://github.com/WhiskeySockets/Baileys/issues/914)

I thought I was the only one who experienced it

**Comment by luispimenta** [comment]
> > Di sini hal yang persis sama terjadi, tetapi dari 40 grup hanya 4 yang memiliki masalah ini, saya dapat mengirim pesan terjadwal atau tidak, dan 4 grup yang sama selalu tidak menerimanya, memberikan kesalahan 428, semua yang lain bekerja dengan sempurna
> > [Masalah saya](https://github.com/WhiskeySockets/Baileys/issues/914)
> 
> I thought I was the only one who experienced it

No, it wasn't, I even met two other people who had the same problem and couldn't solve it, and they went looking for another alternative to Baileys, today they already migrated to another lib, we're trying to find a solution, otherwise the alternative will be to change the Baileys too :(

**Comment by ricco-douro** [comment]
I keep having this error. on v6.6.0. I have upgraded to 6.7.5 and keeps happening.

**Comment by luispimenta** [comment]
Exactly the same here, I've already tried 6.6.0 and the latest 6.7.5 and both happen the same thing

**Comment by NeKosmico** [comment]
I only experience this error when the network is slow or the CPU is loaded too much.

**Comment by KazeDevID** [comment]
> > > Di sini hal yang tetap sama terjadi, tetapi dari 40 grup hanya 4 yang memiliki masalah ini, saya dapat mengirim pesan terjadwal atau tidak, dan 4 grup yang sama selalu tidak tahan, memberikan kesalahan 428, semua yang lain bekerja dengan sempurna 
> > > [Masalah saya](https://github.com/WhiskeySockets/Baileys/issues/914)
> > 
> > 
> > Saya pikir saya satu-satunya yang mengalaminya
> 
> Tidak, bukan itu masalahnya. Saya bahkan bertemu dengan dua orang lain yang mengalami masalah serupa dan tidak dapat menyelesaikannya. Mereka pun mencari alternatif lain untuk Baileys. Hari ini, mereka sudah pindah ke lib lain. Kami sedang berusaha mencari solusinya. Kalau tidak, alternatifnya adalah mengganti Baileys juga :(

keep it up bro

**Comment by xbal023** [comment]
> ![Screenshot_20240716-150345](https://private-user-images.githubusercontent.com/101078139/348976982-4aef8576-849e-40ec-9187-b7f342668eb0.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyNzIzMzYsIm5iZiI6MTcyMTI3MjAzNiwicGF0aCI6Ii8xMDEwNzgxMzkvMzQ4OTc2OTgyLTRhZWY4NTc2LTg0OWUtNDBlYy05MTg3LWI3ZjM0MjY2OGViMC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxOFQwMzA3MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYWFiZTlmYzg4OTAwNjZhNGY4MjRmNjYwNWMyZjEzYWRlNDBjNjY3OTVlMTkxMDVkOWEwMTMxZWM4YWU5ODY3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.N9R-YVhpNQL0sYAt_yxUANh8ojuJL6f-XWs-y4ClxWo)

Tips before opening an issue:
- fix your code first.  
- Make sure the error that occurs is not from your code. 
- If you are still experiencing problems, make sure you - always read and open the DOCUMENTATION.  
- If you don't know the documentation link, please go to this link: [https://baileys.whiskeysockets.io/modules.html](baileys docs)

**Comment by ecatugy** [comment]
I notice this when I'm in debugger mode; it always happens then. However, when compiled and running on the server, it's quite rare.

**Comment by hacxk** [comment]
Try end the socket every 30 Minutes or lower if you prefer and add a reconnect mechanism to auto reconnect

```js

await sock.end()

```

**Comment by nubinter** [comment]
is there a fix for this? i got error connection close too when tried sending to my own number.

**Comment by Paul181516** [comment]
> **same** 428 while sending message to a group works fine in dm

Same here, were you able to fix it?

**Comment by canove** [comment]
try  #1368
